src/pages/frequent_questions.ts

```diff
@@ -1,19 +1,22 @@
 import 'core-js/stable';
 import { createVueApp } from '@src/createVueApp';
-import PageDataInitializer from '@src/page_data_initializer';
+import PageDataInitializer from '@src/util/page_data_initializer';
 import { faqContentFromObject } from '@src/view_models/faq';
 import App from '@src/components/App.vue';
 import Faq from '@src/components/pages/Faq.vue';
-import { createNullFeatureFetcher } from '@src/FeatureFetcher';
+import { createFeatureFetcher } from '@src/util/FeatureFetcher';
+import { bucketIdToCssClass } from '@src/util/bucket_id_to_css_class';
 
 const PAGE_IDENTIFIER = 'faq-page';
 const pageData = new PageDataInitializer<any>( '#appdata' );
+const featureFetcher = createFeatureFetcher( pageData.selectedBuckets, pageData.activeFeatures );
 
-createVueApp( App, pageData.messages, createNullFeatureFetcher(), {
+createVueApp( App, pageData.messages, featureFetcher, {
 	assetsPath: pageData.assetsPath,
+	bucketClasses: bucketIdToCssClass( pageData.selectedBuckets ),
 	pageIdentifier: PAGE_IDENTIFIER,
 	page: Faq,
 	pageProps: {
-		content: faqContentFromObject( pageData.applicationVars ),
+		content: faqContentFromObject( JSON.parse( pageData.applicationVars.faq_content ) ),
```

Comment:
Now I remember why we had custom templates in the PHP application - we wanted to avoid double JSON encoding. Ideally, this line would be 
```suggestion
		content: faqContentFromObject( pageData.applicationVars.faq_content ),
```

But we can live with the slight increase in size and general uglyness for now